src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/activities/GetAllocationResourceIntTest.kt

```diff
@@ -212,13 +212,11 @@ class GetAllocationResourceIntTest : IntegrationTestBase() {
     }
 
     /*
-     * Note that each test in this class also checks the endpoint `/activities/ids`, `/activities/rates-with-unknown-incentives`,
-     *  `/allocations/suspended`, `/allocations/missing-pay-bands` as well as the allocation ids endpoint.
+     * Note that each test in this class also checks the endpoints `/allocations/ids`, `/allocations/suspended` and
+     * `/allocations/missing-pay-bands`.
      *
-     * This is to test that the same rules are being applied to the activities, suspended allocations, allocations with
-     * missing pay rates and allocation ids selection and effectively tests that the following custom queries are aligned:
-     * - CourseActivityRepository.findActiveActivities
-     * - CourseActivityRepository.findRatesWithUnknownIncentive
+     * This is to test that the same rules are being applied to active allocations, suspended allocations and allocations
+     * with missing pay rates and effectively tests that the following custom queries are aligned:
      * - OffenderProgramProfilesRepository.findActiveAllocations
      * - OffenderProgramProfilesRepository.findSuspendedAllocations
      * - OffenderProgramProfilesRepository.findMissingPayBands
```

Comment:
Previously a course activity was included in the migration if it had an active allocation. This meant the rules for activities and allocations were the same, so these tests checked that the same allocations were being included in /activities/ids, /allocations/ids and any derived preview checks SQLs.

This rule no longer holds - activities can now be migrated even if they have no active allocations - so all of the checks on activities endpoints in this class have been removed and we only check the allocations endpoints.